src/sim/fd_array.hh

```diff
@@ -36,9 +36,9 @@
 #include <map>
 #include <memory>
 #include <string>
-
```

Comment:
There should be a new line here. Please re-add it.

Reply:
I re-added it. Thanks for reviewing this!